lib/ballot.go

```diff
@@ -361,10 +384,10 @@ func (b *BallotBoxes) AddBallot(ballot Ballot) (isNew bool, err error) {
 	}
 
 	// unknown ballot will be in `WaitingBox`
-	if ballot.State() == sebakcommon.BallotStateINIT {
-		err = b.AddVotingResult(vr, b.WaitingBox)
-	} else {
+	if ballot.IsFitInVoting() {
 		err = b.AddVotingResult(vr, b.VotingBox)
+	} else {
```

Comment:
I like to write explicit conditions like `if ballot.State () == sebakcommon.BallotStateINIT`. This eliminates the possibility of side effects. 
I am anxious that this code can not filter the case where `b.State ()` returns a garbage value. What about creating an exception for this case?
Of course, this may not be a big problem. this is just my taste.

Reply:
I figure, it is not possible to return a garbage value because `b.State()` only returns BallotState type.
All unexpected or unknown state is BallotStateNONE. Then the ballot should add to WaitingBox, as @spikeekips said.
> // unknown ballot will be in `WaitingBox`